salt/environment_settings.yml

```diff
@@ -136,8 +136,11 @@ environments:
         bucket_uses:
           - grades
           - storage
-          - video-upload
-          - video-delivery
+          - edx-video-upload
+          - edx-video-delivery
```

Comment:
What's the use of the new edx-video-*?

Reply:
Those are for the destinations for edx to upload and play video going to/from the video pipeline.